pkg/kritis/review/review.go

```diff
@@ -21,27 +21,60 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/container"
+	"github.com/grafeas/kritis/pkg/kritis/crd/authority"
 	"github.com/grafeas/kritis/pkg/kritis/crd/securitypolicy"
 	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	"github.com/grafeas/kritis/pkg/kritis/secrets"
 	"github.com/grafeas/kritis/pkg/kritis/util"
 	"github.com/grafeas/kritis/pkg/kritis/violation"
+	containeranalysispb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1alpha1"
 	"k8s.io/api/core/v1"
 )
 
 type Reviewer struct {
-	client   metadata.MetadataFetcher
-	vs       violation.Strategy
+	config    *Config
+	vs        violation.Strategy
+	client    metadata.MetadataFetcher
+	isWebhook bool
+}
+
+type Config struct {
 	validate securitypolicy.ValidateFunc
+	secret   secrets.Fetcher
 }
 
-func New(client metadata.MetadataFetcher, vs violation.Strategy, validate securitypolicy.ValidateFunc) Reviewer {
+func New(client metadata.MetadataFetcher, vs violation.Strategy, isWebhook bool, c *Config) Reviewer {
```

Comment:
In lieu of our presubmit running lint tools, I'll nit about these:

review.go:35:6: exported type Reviewer should have comment or be unexported
review.go:42:6: exported type Config should have comment or be unexported
review.go:47:1: exported function New should have comment or be unexported
review.go:59:1: exported function Default should have comment or be unexported
review.go:66:1: exported function NewConfig should have comment or be unexported